components/aws/sagemaker/batch_transform/README.md

```diff
@@ -4,7 +4,7 @@
 Component to get inferences for an entire dataset in SageMaker from a Kubeflow Pipelines workflow.
 
 ## Details
-With [batch transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html), you create a batch transform job using a trained model and the dataset, which must be stored in Amazon S3. Use batch transform when you:
+With [batch transform](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html), you create a batch transform job using a trained model and the dataset, which must be stored in Amazon S3. Use batch transform when you:
```

Comment:
this link should point to: https://docs.aws.amazon.com/sagemaker/latest/dg/batch-transform.html